lib/developer_portal/app/controllers/developer_portal/admin/account/authorize_net_controller.rb

```diff
@@ -15,19 +15,19 @@ def hosted_success
       auth_response = cim_gateway.get_customer_profile(
```

Comment:
If `auth_response` comes from `cim_gateway`, you might as well instantiate it inside `update_user_and_perform_action!` instead of passing it as argument.

Reply:
In fact, it can all be in the same method 😄